sshd/src/main/scala/ammonite/repl/sshd/util/Implicits.scala

```diff
@@ -0,0 +1,21 @@
+package ammonite.repl.sshd.util
+
+import org.apache.sshd.common.Factory
+import org.apache.sshd.server.{CommandFactory, Command}
+
+import scala.language.implicitConversions
+
+trait Implicits {
```

Comment:
Why do these need to be implicit? I don't see them used anywhere in the example you showed in the readme, and I only see three call-sites inside the codebase. I think we should inline them.